### ISSUE TITLE
RangeError: No value supplied for attribute href on paste

### ISSUE THREAD

**Issue by tommoor** [outline/rich-markdown-editor#207]
Came through production error reporter:

![image](https://user-images.githubusercontent.com/380914/83040747-7612c680-9ff4-11ea-961a-42a2dbb9c51d.png)

```
RangeError: No value supplied for attribute href
  at computeAttrs(./node_modules/prosemirror-model/dist/index.js:1854:13)
  at computeAttrs(./node_modules/prosemirror-model/dist/index.js:1944:9)
  at create(./node_modules/prosemirror-model/dist/index.js:1956:1)
  at enter(./node_modules/prosemirror-model/dist/index.js:2985:1)
  at addElementByRule(./node_modules/prosemirror-model/dist/index.js:2892:1)
  at addElement(./node_modules/prosemirror-model/dist/index.js:2857:1)
  at addDOM(./node_modules/prosemirror-model/dist/index.js:2801:1)
  at addAll(./node_modules/prosemirror-model/dist/index.js:2930:1)
  at parseSlice(./node_modules/prosemirror-model/dist/index.js:2597:1)
  at parseFromClipboard(./node_modules/rich-markdown-editor/node_modules/prosemirror-view/dist/index.js:2764:1)
  at doPaste(./node_modules/rich-markdown-editor/node_modules/prosemirror-view/dist/index.js:3674:1)
  at handler(./node_modules/rich-markdown-editor/node_modules/prosemirror-view/dist/index.js:3686:1)
  at HTMLDivElement.e.dom.addEventListener.e.eventHandlers.<computed>(./node_modules/rich-markdown-editor/node_modules/prosemirror-view/dist/index.js:3179:10)
```

Sentry: https://sentry.io/organizations/outline-jl/issues/1692273767

**Comment by tommoor** [comment]
Need more info to debug, I'm not able to reproduce. Waiting to see if it crops up again.

**Comment by tommoor** [comment]
closed by 72aeb94
